razorpay-payments.php

```diff
@@ -292,44 +324,40 @@ function check_razorpay_response()
 
                 $success = false;
                 $error = "";
+                $status = "";
 
                 $api = new Api($key_id, $key_secret);
-
+                
                 try
                 {
                     if ($this->payment_action === 'authorize')
                     {   
                         $payment = $api->payment->fetch($razorpay_payment_id);
             
-                        $expectedStatus = 'authorized';
+                        $status = 'authorized';
                     }
                     else
                     {
-                        $payment = $api->payment->fetch($razorpay_payment_id);
-                        $amount = $payment->amount;
-                        
-                        $payment = $payment->capture(array('amount'=>$amount)); 
+                        $razorpay_order_id = $woocommerce->session->get('razorpay_order_id');
+                        $razorpay_signature = $_REQUEST['razorpay_signature'];
```

Comment:
- use $_POST over $_REQUEST here. Very slight advantage in terms of knowing where the input is coming from
- https://stackoverflow.com/questions/1149118/does-request-have-security-problem

Maybe implement a validation for the input values here.